lib/components/dashboard/degree-info/Reviewers/CurrentReviewers.tsx

```diff
@@ -1,15 +1,67 @@
-import { CheckIcon } from '@heroicons/react/outline';
+import { CheckIcon, BellIcon } from '@heroicons/react/outline';
 import { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
 import ReactTooltip from 'react-tooltip';
 import { ReviewRequestStatus } from '../../../../resources/commonTypes';
 import { userService } from '../../../../services';
 import { selectPlan } from '../../../../slices/currentPlanSlice';
+import { toast } from 'react-toastify';
+import emailjs from '@emailjs/browser';
+import { getAPI } from '../../../../resources/assets';
 
 const CurrentReviewers = () => {
   const currentPlan = useSelector(selectPlan);
   const [jsx, setJsx] = useState([]);
 
+  const sendEmail = (fromName, toEmail, toName, reviewID) => {
+    var form = {
+      from_name: fromName,
+      to_email: toEmail,
+      to_name: toName,
+    };
+
+    const body = {
+      review_id: reviewID,
+      status: 'UNDERREVIEW',
+    };
+
+    fetch(getAPI(window) + '/planReview/changeStatus', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(body),
+    })
+      .then((response) => {
+        if (response.status === 200) {
+          // Status successfully changed to UNDERREVIEW
+          emailjs
```

Comment:
Wait so why are emails being sent in the frontend

Reply:
because we haven't implemented emails being sent in the backend yet. this will be looked at in a later iteration... or sometime later when higher priority implementations are completed.